Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -1269,7 +1269,8 @@ EditorState WebPage::editorState(ShouldPerformLayout shouldPerformLayout) const
         document->updateLayout(); // May cause document destruction
 
     if (auto* frameView = document->view(); frameView && !frameView->needsLayout()) {
-        result.isMissingPostLayoutData = false;
+        result.ensurePostLayoutData();
+        result.ensureVisualData();
 
         auto& postLayoutData = result.postLayoutData();
```

Comment:
Nit - instead of `result.ensurePostLayoutData();` above, it might be cleaner to just do:

```
auto& postLayoutData = result.ensurePostLayoutData();
…
```

Reply:
Definitely.